packages/fxa-auth-server/lib/payments/paypal/helper.ts

```diff
@@ -35,10 +35,12 @@ type PaypalHelperOptions = {
 };
 
 type AgreementDetails = {
-  status: 'active' | 'cancelled';
   city: string;
   countryCode: string;
+  firstName: string;
+  lastName: string;
   state: string;
```

Comment:
What's the difference between [Payer name fields](https://developer.paypal.com/api/nvp-soap/ba-update-nvp/#link-payernamefields) `FIRSTNAME` and `LASTNAME` and [Bill to Address Fields](https://developer.paypal.com/api/nvp-soap/ba-update-nvp/#link-billtoaddressfields) `BILLINGNAME`?